schema/migration-4-0003-20210116.sql

```diff
@@ -26,6 +26,7 @@ CREATE INDEX IF NOT EXISTS idx_tx_in_tx_in_id ON tx_in(tx_in_id) ;
 CREATE INDEX IF NOT EXISTS idx_tx_in_tx_out_id ON tx_in(tx_out_id) ;
 CREATE INDEX IF NOT EXISTS idx_tx_metadata_tx_id ON tx_metadata(tx_id) ;
 CREATE INDEX IF NOT EXISTS idx_tx_out_tx_id ON tx_out(tx_id) ;
+CREATE INDEX IF NOT EXISTS idx_tx_cbor_tx_id ON tx_cbor(tx_id) ;
```

Comment:
It might be that you have to generate a new migration file using [these instructions](https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/schema-management.md#creating-a-migration)

Reply:
Added as separate file and updated counts in `cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Conway/CommandLineArg/ForceIndex.hs` as well.